src/server/enrollment-system.js

```diff
@@ -341,9 +341,6 @@ function childToDependentInfo(child) {
  */
 function childToDependentFinancialsInfo(child) {
   const incomes = resourceToIncomeCollection(child);
-  if (!incomes) {
```

Comment:
I'm wondering if this is sufficient, or if we need to include some logic in `veteranToFinancialsInfo` that only includes spouse and dependent financial info if veteran income info also exists? It sounded like from Josh that we should only include spouse and dependent info if they provide veteran income information, but the logic in `veteranToFinancialsInfo` looks for any of them being present? Will this mean that if there is no veteran income, but they do enter spouse information, that it will be included? I don't think it should be in that case.

```
const hasIncomeData = expenses || incomes || hasSpouseIncome || hasDependentFinancials;

if (!hasIncomeData) {
  return undefined;
}
```